index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+   <head>
+      <meta charset="utf-8">
+      <title>Adopt-a-Dog homepage: Look here to learn more about Seattle's premiere dog adoption agency</title>
+      <link rel="stylesheet" type="text/css" media="only screen and (min-width: 481px)" href="stylesheets/desktop/stylesheet-index.css">
+      <link rel="stylesheet" type="text/css" media="only screen and (max-width: 480px)" href="stylesheets/mobile/stylesheet-index-mobile.css">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+   </head>
+   <body>
+      <header>
+        <div id="header-checkout-all">
+         <div class="header-checkout">
+            <img class="cart-logo" src="images/cart-icon.svg" alt="Shopping cart icon">
+            <p id="header-price">$1234.56</p>
+          </div>
+            <a class="button" href="checkout.html">Checkout</a>
+            <div class="header-intro">
+            <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo">
+            <h2 class="header-title"><em>Seattle's premiere dog adoption agency</em></h2>
+         </div>
+       </div>
+      </header>
+      <nav class="global-navigation">
+         <a href="index.html" class="nav-link">Home</a>
+         <p> / </p>
+         <a href="dogs.html" class="nav-link-2">Dogs</a>
+         <p> / </p>
+         <a href="blog.html" class="nav-link-2">Blog</a>
+      </nav>
+      <section class="index-content">
+         <div class="index-1">
+            <img id="index-image" src="images/home-intro.jpg" alt="A picture of a man hugging a dog; the dog is smiling and looking into the camera">
+         </div>
+         <div class="index-2">
```

Comment:
In fact I might consider changing the html slightly to have a div container, then the img and the text as siblings. That way you could have the img take half the page by using width=50% and the text will automatically be side by side and wrap around underneath the img if the page becomes too small. ie:
`<div >`
`  <img>`
` . <p> some text...</p>`
`</div>`